src/chart_types/xy_chart/rendering/points.ts

```diff
@@ -228,3 +235,26 @@ export function getRadiusFn(
     return circleRadius ? Math.sqrt(circleRadius + baseMagicNumber) + lineWidth : lineWidth;
   };
 }
+
+function yAccessorForOrphanCheck(datum: DataSeriesDatum): number | null {
+  return datum.filled?.y1 ? null : datum.y1;
+}
+
+function isOrphanData(
```

Comment:
maybe a little more specific?

```suggestion
function isOrphanDataPoint(
```

Reply:
done https://github.com/elastic/elastic-charts/pull/900/commits/b94d979f545839e78bbcd2eab59a421346ccea1c